perception/multi_object_tracker/src/data_association/successive_shortest_path/successive_shortest_path.cpp

```diff
@@ -47,10 +47,6 @@ void SSP::maximizeLinearAssignment(
   const std::vector<std::vector<double>> & cost, std::unordered_map<int, int> * direct_assignment,
   std::unordered_map<int, int> * reverse_assignment)
 {
-  // NOTE: Need to set as default arguments
```

Comment:
With this statement, I think it's better to define sparse_cost in the arguments and give default true value.

Reply:
Thank you for your advice!! I will accepr your suggestion.